test/csuite/tiered_abort/main.c

```diff
@@ -131,11 +131,6 @@ typedef struct {
     uint32_t info;
 } THREAD_DATA;
 
-/*
- * TODO: WT-7833 Lock to coordinate inserts and flush_tier. This lock should be removed when that
```

Comment:
There are other csuite tests with this identical usage (possibly wtperf also). They should be fixed. test/schema_abort, test/random_directio. I had opened a ticket to clean those up after in WT-9425. But feel free to do it here and mark that a duplicate.